README.md

```diff
@@ -75,16 +80,20 @@ where `~` is your user's home directory.
 
 3. Build a Docker image for the project.
     ```sh
-    docker build -t unizin-validation .
+    docker compose build
     ```
 
-4. Run a container using the just-created image and two volume mounts for input and output,
-replacing `{absolute_path}` with the absolute path to your home directory.
+4. Run one of the job services
     ```sh
-    docker run \
-        --mount type=bind,source=/{absolute_path}/secrets/unizin-validation,target=/app/config \
-        --mount type=bind,source=/{absolute_path}/data/unizin-validation,target=/app/data \
-        unizin-validation
+    # For the UDW job
```

Comment:
I'm guessing the "UDW job" is just UDW while "Unizin job" is both jobs? I could look further in the code but it would be good to clarify this a little more in the README. 

Is there any way to just run the "UDP job"? We may eventually just want to have it so you can pass the specific job names you want to run on the CLI and have them documented what they are to further reduce the need of needing to change this code. But that could be a new feature issue.

Reply:
There's no UDP job, but it's easy enough to add that. So far there is only one UDP query. I'll add a note about the job definitions, and I can add a UDP job.

Reply:
You can pass the job names as part of a CLI, I just didn't know how to make those args get passed from Docker Compose. So I went the multiple service route (https://github.com/tl-its-umich-edu/unizin-validation/pull/33#discussion_r1090814915), since that was working for me.

Reply:
Ah cool, so we already did this and this is more for convenience. 😀

It looks like with docker compose run you can override it similar to docker run. It seems only way to easily do that with docker compose up is with environment variables. 

https://www.baeldung.com/linux/docker-cmd-override#overriding-cmd-with-the-docker-compose-run-command

Reply:
Added UDP job and more info to README. https://github.com/tl-its-umich-edu/unizin-validation/pull/33/commits/968961a4fbb2a732ae13b6d15ede880e12e9dc6f